src/apicast-cli/cmd/generator.lua

```diff
@@ -13,6 +13,10 @@ local create_new_project = function (args, _)
   local command = {"cp", "-a", raw_app, new_app}
   print('Generating new project into ' .. new_app)
   assert(os.execute(table.concat(command, " ")))
+  
+  print('Generating rockspec file')
+  os.execute('luarocks write-rockspec --output ' .. new_app .. ' ' .. name .. ' 0.1 ' .. new_app)
```

Comment:
You can use "luarocks.write_rockspec" directly

https://github.com/luarocks/luarocks/blob/master/src/luarocks/write_rockspec.lua

Use the `write_rockspec.command`